bot/exts/advent_of_code/_cog.py

```diff
@@ -410,22 +416,27 @@ async def aoc_leaderboard(self, ctx: commands.Context, *, aoc_name: str | None =
         await ctx.send(content=f"{header}\n\n{table}", embed=info_embed)
         return
 
+    @in_month(Month.DECEMBER, Month.JANUARY, Month.FEBRUARY)
+    @adventofcode_group.command(
+        name="changes",
+        brief="Frequently Asked Questions about Advent of Code changes in 2025",
+    )
+    @in_whitelist(channels=AOC_WHITELIST_RESTRICTED, redirect=AOC_REDIRECT)
+    async def aoc_changes(self, ctx: commands.Context) -> None:
+        """Get answers to frequently asked questions about Advent of Code changes in 2025."""
+        await ctx.send(embed=self.cached_changes)
+
```

Comment:
This should be removed. Message in FAQ channel will address this in addition to the 2 announcements that will go out. Don't want to maintain this in future years.